content/docs/apis.md

```diff
@@ -2,57 +2,195 @@
 title: APIs
 description: Interact with TiKV using the raw key-value API or the transactional key-value API
 weight: 4
-draft: true
+draft: false
 ---
 
-TiKV offers two APIs that you can use to interact with it:
+TiKV offers two APIs that you can interact with:
 
-* A [raw](#raw) API
-* A [transactional](#transactional) API
+API | Description | Atomicity | Use when...
+:---|:------------|:----------|:-----------
+[Raw](#raw) | A lower-level key-value API for interacting directly with individual key-value pairs. | Single key | Your application doesn't require distributed transactions or multi-version concurrency control (MVCC)
+[Transactional](#transactional) | A higher-level key-value API that provides ACID semantics | Multiple keys | Your application requires distributed transactions and/or MVCC
+
+{{< info >}}
+It is **not recommended or supported** to use both the raw and transactional APIs on the same keyspace.
+{{< /info >}}
+
+There are several clients that connect to TiKV:
+
+* [Rust](https://github.com/tikv/client-rust)
+* [Java](https://github.com/tikv/client-java)
+* [Go](https://github.com/pingcap/tidb/store/tikv)
+
+Below we use the Rust client for some examples, but you should find all clients work similarly.
+
+## Connecting a Client
+
+This guide assumes you are using Rust and have Rust (>=1.31) installed through `rustup` or other means. You will also need an already deployed TiKV and PD cluster, since TiKV is not an embedded database.
```

Comment:
" through `rustup` or other means." Since it's not limited to `rustup`, we may want to save this info.
```suggestion
This guide assumes you are using Rust 1.31 or above. You will also need an already deployed TiKV and PD cluster, since TiKV is not an embedded database.
```

Reply:
Nice catch!